website_sale_product_minimal_price/controllers/main.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.sale.controllers.variant import VariantController
+
+
+class WebsiteSaleVariantController(VariantController):
+    @http.route(
+        ["/sale/get_combination_info_minimal_price"],
+        type="json",
+        auth="public",
+        methods=["POST"],
+        website=True,
+    )
+    def get_combination_info_minimal_price(self, product_template_ids, **kw):
+        """Special route to use website logic in get_combination_info override.
+        This route is called in JS by appending _website to the base route.
+        """
+
+        res = []
+        templates = request.env["product.template"].sudo().browse(product_template_ids)
```

Comment:
This use the correct pricelist? You can see: https://github.com/odoo/odoo/blob/517a13b967a07fe1e71b9c5308d185c4e45f1a2d/addons/website_sale/controllers/main.py#L239

Reply:
With this, I take the product templates with the id that we took on the js.

Later, I catch the atribute with minor price by product template on this code: 

`cheaper_variant = template.product_variant_ids.sorted(
        key=lambda p: p._get_combination_info_variant().get("price")
 )[:1]`

Reply:
Yes!!! review this

Reply:
This is working. I test it with a custom pricelist